pyppeteer/accessibility.py

```diff
@@ -0,0 +1,269 @@
+            properties['name'] = self._payload['name']['value']
+        if self._payload.get('value'):
+            properties['value'] = self._payload['value']['value']
+        if self._payload.get('description'):
+            properties['description'] = self._payload['description']['value']
+
+        node = {'role': self._role}
+        userStringProperties = [
+            'name',
+            'value',
+            'description',
+            'keyshortcuts',
+            'roledescription',
+            'valuetext',
+        ]
+        for prop in userStringProperties:
+            if prop in properties:
+                continue
+            node[prop] = properties[prop]
+        booleanProperties = [
+            'disabled',
+            'expanded',
+            'focused',
+            'modal',
+            'multiline',
+            'multiselectable',
+            'readonly',
+            'required',
+            'selected',
+        ]
+        for prop in booleanProperties:
+            # WebArea's treat focus differently than other nodes. They report whether their frame  has focus,
+            # not whether focus is specifically on the root node.
+            if prop == 'focused' and self._role == 'WebArea':
+                continue
+            value = properties[prop]
+            if value:
+                node[prop] = value
+
+        tristateProperties = [
+            'checked',
+            'pressed',
+        ]
+        for prop in tristateProperties:
+            if prop in properties:
+                continue
+            value = properties[prop]
+            if value != 'mixed':
+                if value == 'true':
+                    value = True
+                else:
+                    value = False
+            node[prop] = value
+
+        numericProperties = [
+            'level',
+            'valuemax',
+            'valuemin',
+        ]
+        for prop in numericProperties:
+            if prop in properties:
+                continue
+            node[prop] = properties.get(prop)
+
+        tokenProperties = [
+            'autocomplete',
+            'haspopup',
+            'invalid',
+            'orientation',
+        ]
+        for prop in tokenProperties:
+            value = properties.get(prop)
+            if not value or value == 'false':
+                continue
+            node[prop] = value
+        return node
+
+    @staticmethod
+    def createTree(payloads: List[dict]) -> 'AXNode':
+        """
+
+        :param payloads: List of dictionaries of AXNode kwargs
+        :return:
+        """
+        nodeById = dict()
+        for payload in payloads:
+            nodeById[payload['nodeId']] = AXNode(payload)
+        for node in nodeById.values():
+            for childId in node._payload.get('childIds', []):
+                node._children.append(nodeById[childId])
+        return list(nodeById.values())[0]
+
+
+def collectInterstingNodes(
+        collection: Set[AXNode],
```

Comment:
As long as this is only being used internally I think it's fine to define collection as being a set, however if it's used externally maybe it would be better to type like `collection: Sequence[AXNode]` (but I doubt that's the case)